src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -0,0 +1,68 @@
+package analytics
+
+import (
+	"net/http"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/getlantern/analytics"
+	"github.com/getlantern/flashlight/ui"
+	"github.com/getlantern/golog"
+)
+
+const (
+	messageType = `Analytics`
+)
+
+var (
+	log        = golog.LoggerFor("flashlight.analytics")
+	service    *ui.Service
+	httpClient *http.Client
+)
+
+func Configure(autoReport *bool, newClient *http.Client) {
+	httpClient = newClient
+
+	if autoReport != nil && *autoReport {
+		err := Start()
+		if err != nil {
+			log.Errorf("Error starting analytics service: %q", err)
+		}
+	}
```

Comment:
Yes, I think that would be nice if it's easy enough to do reliably.  That said, what resources are we talking about?  A socket and a few lines of code that run on every page view?  Actually, now that we're talking about it, it looks like we just track page view when the UI is opened, right?  Do we even need a websocket service for this, or would a REST endpoint suffice?